addons/mrp/views/mrp_production_views.xml

```diff
@@ -121,7 +121,7 @@
                                     <field name="needs_lots" readonly="1" groups="stock.group_production_lot"/>
                                     <field name="is_done" invisible="1"/>
                                     <field name="sequence" invisible="1"/>
-                                    <field name="location_id" domain="[('id', 'child_of', parent.location_id)]" invisible="1"/>
+                                    <field name="location_id" invisible="1"/>
```

Comment:
Are you sure you can just delete the domain? Maybe the field has been renamed? I see `production_location_id` on the parent, could it be it?

Reply:
There are 3 fields referring to locations in the parent model...

Reply:
We don't care since it's invisible